src/CMake/findpackage.cmake

```diff
@@ -0,0 +1,60 @@
+message("-- Preparing XRT find_package")
+
+include(CMakePackageConfigHelpers)
+
+string(TOLOWER ${PROJECT_NAME} LOWER_NAME)
+
+file(RELATIVE_PATH REL_INCLUDE_DIR
+  ${XRT_INSTALL_DIR}/share/cmake/${PROJECT_NAME}
+  ${XRT_INSTALL_DIR}/include
+  )
+
+set(CONF_INCLUDE_DIRS "\${${PROJECT_NAME}_CMAKE_DIR}/${REL_INCLUDE_DIR}")
```

Comment:
Does this mean that when building source code 'share/cmake/...' is added to the include search path? I see CONF_INCLUDE_DIRS used in xrt.fp.in as XRT_INCLUDE_DIRS?

Reply:
xrt.fp.in is a template file that gets filled in by the call to "configure_file". conf_include_dirs is used to populate XRT_INCLUDE_DIRS.

After configuration happens, the include is set relative to the path where the cmake file is installed.

This is what it is in my test:

```
/opt/xilinx/xrt/share/cmake/XRT/../../../include
```

To be honest... I copied this (INCLUDE_DIR) code from another example... If I put more brain power into it... I might be able to make it simpler...

I don't see why I couldn't use `${XRT_INSTALL_DIR}/include`